spec/octopus/serverspec/type/octopus_deploy_account_spec.rb

```diff
@@ -20,8 +20,8 @@
         to raise_error(/account_name/)
   end
 
-  ex_does_not_support_spaces = File.read('spec/octopus/serverspec/json/apidoesnotsupportspaces.json')
-  ex_acc_found_response = File.read('spec/octopus/serverspec/json/accountsall.json')
+  ex_does_not_support_spaces = get_api_example('/api/2018.4.5')
```

Comment:
Having `2018.4.5` and `2019.4.5` could cause confusion (the last 2 parts being the same).
Might be better if it was a diff version on one side?

Reply:
Easy enough to change the filename and/or the version